doc/cephfs/mds-journaling.rst

```diff
@@ -141,15 +141,6 @@ The targetted size of a log segment in terms of number of events is controlled b
 
 .. confval:: mds_log_events_per_segment
 
-The frequency of major segments (noted by the journaling of the latest ``ESubtreeMap``) is controlled by:
-
-.. confval:: mds_log_major_segment_event_ratio
-
-When ``mds_log_events_per_segment * mds_log_major_segment_event_ratio``
-non-``ESubtreeMap`` events are logged, the MDS will journal a new
-``ESubtreeMap``. This is necessary to allow the journal to shrink in size
-during the trimming of expired segments.
```

Comment:
Why not update this with the new `mds_log_minor_segments_per_major_segment`?

Reply:
Not sure why I didn't add that....